### PR TITLE
added templates for SCP

### PR DESCRIPTION
**Pull request by Krishnan-Priyanshu**
##### SUMMARY

Relates OR Closes #0000

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

#### Release note for [CHANGELOG](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md):

```release-note

```

#### Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

#### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

### CODE REVIEW

**Comment by terraform-providers-svc** [comment]
Can one of the admins verify this patch?

**Review by Sakshi-dell — DISMISSED** [review]

File: templates/resources/idrac_firmware_update.md.tmpl [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+terraform.tfvars
+{{ tffile ( printf "examples/resources/%s/terraform.tfvars" .Name ) }}
+
+provider.tf
+{{ tffile ( printf "examples/resources/%s/provider.tf" .Name ) }}
+
+main.tf
+{{tffile .ExampleFile }}
+
+After the successful execution of the above resource block, iDRAC System attributes configuration would have been altered. It can be verified through state file.
```
Comment (Sakshi-dell) [inline templates/resources/idrac_firmware_update.md.tmpl:44]: system attributes need to be updated to firmware attributes

File: templates/resources/idrac_server_configuration_profile_import.md.tmpl [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+terraform.tfvars
+{{ tffile ( printf "examples/resources/%s/terraform.tfvars" .Name ) }}
+
+provider.tf
+{{ tffile ( printf "examples/resources/%s/provider.tf" .Name ) }}
+
+main.tf
+{{tffile .ExampleFile }}
+
+After the successful execution of the above resource block, Server Configuration Profile will be exported to share type.
```
Comment (Sakshi-dell) [inline templates/resources/idrac_server_configuration_profile_import.md.tmpl:44]: exported -> imported

File: docs/guides/using_export_and_import_scp.md [hunk 3]
```diff
@@ -0,0 +1,277 @@
+---
+# Copyright (c) 2023-2024 Dell Inc., or its subsidiaries. All Rights Reserved.
```
Comment (rounak-adhikary) [inline docs/guides/using_export_and_import_scp.md:2]:
```suggestion
# Copyright (c) 2024 Dell Inc., or its subsidiaries. All Rights Reserved.
```

**Review by rounak-adhikary — APPROVED** [review]

**Review by Sakshi-dell — APPROVED** [review]

**Pull request merged**